greengrass_parsec_workshop/build_demo.sh

```diff
@@ -28,12 +30,14 @@ function dirty_build_on_new_comits() {
 }
 
 function build_greengrass_patched() {
-pushd ./aws-greengrass-parsec-provider/parsec-greengrass-run-config/docker/
-docker build . \
-       --build-arg BUILD_TS=${DIRTY_TS} \
-       --tag parallaxsecond/greengrass_patched:latest \
-       --progress plain
-popd
+  echo "Build greengrass patched ..."
+  pushd ./aws-greengrass-parsec-provider/parsec-greengrass-run-config/docker/
+  docker build . \
+         --build-arg BUILD_TS=${DIRTY_TS} \
+         --tag parallaxsecond/greengrass_patched:latest \
+         --progress plain
+  popd
+  echo "... greengrass patched successfully built."
 }
 
 function copy_deps_from_greengrass_patched_to_local() {
```

Comment:
Why do we still use this function?  The local repository is not used for the demo build. I think it's left from some dev testing.